mm2src/coins/lp_coins.rs

```diff
@@ -4753,6 +4846,12 @@ pub async fn my_tx_history(ctx: MmArc, req: Json) -> Result<Response<Vec<u8>>, S
     Ok(try_s!(Response::builder().body(body)))
 }
 
+/// `get_trade_fee` rpc implementation.
+/// There is some consideration about this rpc:  
+/// for eth coin this rpc returns max possible trade fee (estimated for maximum possible gas limit for any kind of swap).
+/// However for eth coin, as part of fixing this issue https://github.com/KomodoPlatform/komodo-defi-framework/issues/1848,
+/// `max_taker_vol' and `trade_preimage` rpc now return more accurate required gas calculations.
+/// So maybe it would be better to deprecate this `get_trade_fee` rpc
```

Comment:
Add comment about this in documentation of changes for QA. I think `trade_preimage` and `max_taker_vol` are used currently instead of this but to make sure.

Reply:
added (below https://github.com/KomodoPlatform/komodo-defi-framework/pull/2051#issuecomment-2118192179)